README.md

```diff
@@ -106,22 +106,23 @@ When building C/C++ software, often the build is performed in parallel via a com
 If you are in an Ubuntu docker container, omit `sudo` from all commands because you run as `root` by default. Most other docker containers also exclude `sudo`, especially Debian-family containers. If your shell prompt is a hash tag (`#`), omit `sudo`.
 
 #### Pinned Build
-Make sure you are in the root of the `leap` repo, then run the `install_depts.sh` script to install dependencies:
-```bash
-sudo scripts/install_deps.sh
-```
-
-Next, run the pinned build script. You have to give it three arguments in the following order:
+Make sure you are in the root of the `leap` repo, then run the pinned build script. You have to give it three arguments in the following order:
 1. A temporary folder, for all dependencies that need to be built from source.
 1. A build folder, where the binaries you need to install will be built to.
 1. The number of jobs or CPU cores/threads to use (note the [jobs flag](#step-3---build) warning above).
 
-> 🔒 You do not need to run this script with `sudo` or as root.
+> 🔒 You do not need to run this script with `sudo` or as root. On Debian-family operating systems such as Ubuntu, the script will determine if all package dependencies are installed and only invoke `apt-get` if you are missing packages. In that case, you may be prompted for your password if `apt-get` needs additional permissions from `sudo`. You should not run this script with `sudo` or as root because you do not need to.
 
 For example, the following command runs the `pinned_build.sh` script, specifies a `deps` and `build` folder in the root of the Leap repo for the first two arguments, then builds the packages using all of your computer's CPU threads:
 ```bash
 scripts/pinned_build.sh deps build "$(nproc)"
 ```
+If you want to by-pass the `[Y/n]` prompt from `apt-get install`, you can pass `-y` to the script like this.
+```bash
+echo '-y' | scripts/pinned_build.sh deps build "$(nproc)"
```

Comment:
Not sure how that works. `sudo` doesn't consume from stdin? I don't think `-y` is correct here anyways, from this test
```
echo -y | apt-get upgrade
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  base-files libgmp10 libgnutls30 libpam-modules libpam-modules-bin libpam-runtime libpam0g libpcre2-8-0
  libsystemd0 libudev1 login passwd perl-base tar zlib1g
15 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 4893 kB of archives.
After this operation, 14.3 kB of additional disk space will be used.
Do you want to continue? [Y/n] Abort.
```

It seems like you may want to allow the user to pass `-y` as a script argument, and then forward that through to apt-get